### PR TITLE
webpack style-loader

### LINKED ISSUE #195: Find a better way to inject mocha CSS
The mocha-CSS.js file contains a very long and ugly string with the mocha CSS. Not sure why it is done like this, but assume it needs to be dynamically injected or something like that. We should see if we can find a more maintainable way of doing this. Hopefully via a normal css file with a .css extension?

### PR DESCRIPTION
**Pull request by tbushman**
#### Pre-Submission Checklist

- [x] Your pull request targets the `master` branch of freeCodeCamp/testable-projects-fcc
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] Your changes have been tested either locally or using a newly created CDN based on your fork's testable-projects-fcc/build/bundle.js file
- [x] Working changes have been added to the build by running `npm run build`

#### Type of Change

- [ ] Small bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.

- [x] Closes currently open issue: Closes #195

#### Description
  

Here's one way using webpack style-loader and css-loader modules. ...

### CODE REVIEW

**Comment by tchaffee** [comment]
@tbushman I am ready to look at this now, but I want to check out the project locally because it's easier to look at the code in full in my editor, rather than looking at the changes here. Could you please rebase this over master and then I'll take a look?

**Comment by tbushman** [comment]
@tchaffee okiedokie

File: src/index.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import $ from 'jquery';
 import chai from 'chai';
 import testSuiteSkeleton from './assets/test-suite-skeleton';
-import mochaCSS from './assets/mocha-CSS';
+import mochaCSS from './stylesheets/style.css';
```
Comment (tchaffee) [inline src/index.js]: As we discussed via Skype, maybe this variable could have a more accurate name than `mochaCSS`? Something like `testSuiteFCCStyles`?

File: webpack.config.dev.js [hunk 2]
```diff
@@ -18,6 +18,11 @@ module.exports = {
                 query: {
                     presets: ['es2015']
                 }
+            },
+            {
+              test: /\.css$/,
+              exclude: /node_modules/,
+              loader: "style-loader!css-loader"
```
Comment (tchaffee) [inline webpack.config.dev.js]: Could you please put an inline comment documenting this line? I sort of think I know what style-loader does, but I read all the docs and could not figure out why you need the `!css-loader` part, or what it does. Also  note you should be using single quotes.

File: webpack.config.js [hunk 3]
```diff
@@ -16,6 +16,11 @@ module.exports = {
                 query: {
                     presets: ['es2015']
                 }
+            },
+            {
+              test: /\.css$/,
+              exclude: /node_modules/,
+              loader: "style-loader!css-loader"
```
Comment (tchaffee) [inline webpack.config.js]: Same as above.

**Comment by tbushman** [comment]
@tchaffee OK, take a look. I found out that with style-loader and css-loader chained in webpack config, the css is appended automatically.  
Also, re: live-server... I still use it, for example, to test the mocha modal.

File: src/index.js [hunk 4]
```diff
@@ -3,7 +3,8 @@
 import $ from 'jquery';
 import chai from 'chai';
 import testSuiteSkeleton from './assets/test-suite-skeleton';
-import mochaCSS from './assets/mocha-CSS';
+import testSuiteFCCStyles from 
+  '!style-loader!css-loader!./stylesheets/style.css';
```
Comment (tbushman) [inline src/index.js]: We still need to import the styles, but we don't need to append them to document.head

Comment (tchaffee) [reply src/index.js]: Could you please put an inline comment about this to explain what it is doing? With a link to documentation would also be helpful.

**Comment by tchaffee** [comment]
@tbushman You should be able to use serve-https instead of live-server? I think you removed the live-server from package.json anyway?

**Comment by tbushman** [comment]
@tchaffee Turns out the !- prefixed import path means specified plugins are used to load the imported file, so that no webpack.config changes would be required. I'm sure you'll check the SO link in comments and see the 'not ideal' characterization of this method, but since we are only loading one stylesheet, this works. One line. :D

**Comment by tchaffee** [comment]
@tbushman I guess what I'm confused about is that the SO answer also gives a way of doing this by specifying the loaders in the webpack config, which is the standard way of doing things. Is there a reason we're not doing it that way? There's no reason IMO to avoid changing the webpack config.

**Comment by tchaffee** [comment]
LGTM

**Pull request merged**